src/vs/workbench/browser/web.api.ts

```diff
@@ -38,6 +38,54 @@ export interface IWorkbench {
 		executeCommand(command: string, ...args: any[]): Promise<unknown>;
 	};
 
+	progress: {
```

Comment:
Hm, I wonder if we can pull this file in versus re-declaring it as the `IEmbedderApi`?

Reply:
Hm I think if we move where the Web API sits this would be possible. Currently it complains about import rules when I tried to import it directly.